Sources/MountebankSwift/Common/JSON.swift

```diff
@@ -24,6 +24,13 @@
 
 import Foundation
 
+fileprivate let encoder: JSONEncoder = {
```

Comment:
Maybe rename `encoder` to `jsonEncoder`? So it is clear that is encoding json. (also for the decoder)

Reply:
That will conflict with https://github.com/MountebankSwift/MountebankSwift/pull/24/files#diff-70b1ef696dc58c79d14276a40475008f9961bb3106b64e57e923ce9ac5a8b25eR5

Reply:
Changed in https://github.com/MountebankSwift/MountebankSwift/pull/24/commits/4af98fcc079aab51aea5c206df5a62b644e681ca